src/Microsoft.VisualBasic/src/Configurations.props

```diff
@@ -2,7 +2,8 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
-      netstandard;
+      netstandard-Unix;
```

Comment:
We want to keep a RID-less implementation since this goes into a package.  Can we rename this configuration from `netstandard-Unix` to just `netstandard`?  Then below give the `FileSystem.Unix.vb` file a different name.